### ISSUE TITLE
href link in Today widget

### ISSUE THREAD

**Issue by mrquatsch** [felixhageloh/uebersicht#160]
First, this may not be the correct place for this type of request. If not, just point me to the appropriate location.

I just discovered this wonderful app/toolset and have been trying to modify my own versions of the widgets that people have produced(getting my feet wet). I'm currently struggling on adding an href link to my modified TodayNews.coffee widget. I have tried testing a few things out and I can either...
1) Get the link url text
2) Get a reference to the object that I'm storing the text within

Ultimately, I want a link on my desktop that allows me to open the link I'm grabbing from the rss feed of my choice. 

Here are my attempts to do so:
https://github.com/mrquatsch/widgets/blob/master/Today.widget/TodaysNews.coffee#L53-L58

```
  ## This one links to 127.0.0.1/%5Bobject%20Element%5D
  #dom.find(link).html("<a href=\"" + theLink + "\">ReadMore</a>") #
  ## Prints the url, but I want to encapsulate it inside of an href
  #dom.find(link).html(theLink) 
  ## This one links back to 127.0.0.1
  #dom.find(link).html("<a href=\"#theLink\">ReadMore</a>")
```

Is there a way to create that link in my widget using the text contained in the "theLink" object?

**Comment by hawkrives** [comment]
What do you see if you print it to the Übersicht debug console with `console.log(link)`? 

The `127.0.0.1/%5Bobject%20Element%5D` looks like it used the text `[object Element]`, which means that there's probably some attribute that actually contains the link text, so printing it in the debug console would let us examine it.

**Comment by mrquatsch** [comment]
That helped! I got it working.

Using http://localhost:41416 I used the "view source" option to run ```console.log(link)```. 

I found a div with a child node that contained my link text. I then updated the path on the "theLink" variable to ```theLink = theItem.find('link')[win.todaysNewsCurrentStory].childNodes[0].textContent```. Doing that, I could correctly get the correct value for my href and was able to correctly create a link on the widget! 

```dom.find(link).html("<a href=\"" + theLink + "\">ReadMore</a>")```

This makes perfect sense from a debugging standpoint. I even took things bit further and created my link using the originating article url(in lieu of the forum link).

Thanks for the guidance @hawkrives !
